src/amo/containers/CategoryPage.js

```diff
@@ -9,6 +9,7 @@ import { apiAddonType } from 'core/utils';
 
 
 export function mapStateToProps(state, ownProps) {
+  const enableSearchSort = false;
```

Comment:
I did that and adjusted the tests (which become ever-so-slightly more verbose) but I like the compositon over overloading `mapStateToProps`, thanks 👍